test/ui/cypress/support/auth.js

```diff
@@ -0,0 +1,51 @@
+import { verifyRequiredEnvVariables } from './../e2e/environment_helpers'
+
+Cypress.Commands.add('loginWithRHAccount', () => {
+   verifyRequiredEnvVariables()
+
+    cy.visit(Cypress.env("DEPLOYMENT_DRIVER_URL"), {
```

Comment:
Let's move this to `cy.beforeEach` in the test suite to make it obvious what the test suite does.